js/libs/form.js

```diff
@@ -416,7 +416,13 @@ Fliplet.Widget.instance('form-builder', function(data) {
             $vm.fields = getFields();
             $vm.isLoading = false;
           }).catch(function (err) {
-            $vm.error = err.message || err.description || err;
+            var error = Fliplet.parseError(err);
+            $vm.error = error;
+            $vm.isLoading = false;
+
+            Fliplet.UI.Toast.error(error, {
+              message: 'The entry to edit could not be loaded.'
```

Comment:
Since it's always about editing entries and users would have likely got here via some UI/interaction that implies "editing", perhaps the "edit" context would be clear to the user and we can simplify the message:

```suggestion
              message: 'Unable to load entry'
```

Thoughts?